lxd/endpoints/endpoints.go

```diff
@@ -67,6 +67,9 @@ type Config struct {
 
 	// HTTP server handling requests for the LXD metrics API.
 	MetricsServer *http.Server
+
+	// HTTP server handling requests from VMs via the vsock.
+	VMSockServer *http.Server
```

Comment:
We should really refer to this as `VsockServer` and `vsock` in general as that's its name.
vmsock is a VMWare equivalent technology.